### PR TITLE
Fix issue 236

### PR DESCRIPTION
**Pull request by komirad**
Resubmit with tests

FYI, executing phpunit gave me 2 failures.

```
1) Google\AdsApi\Common\SoapLogMessageFormatterTest::testFormatDetailed
Failed asserting that two strings are identical.
...
\googleads-php-lib\tests\Google\AdsApi\Common\SoapLogMessageFormatterTest.php:128

2) Google\AdsApi\Common\SoapLogMessageFormatterTest::testFormatDetailedWithScrubbing
Failed asserting that two strings are identical.
...
\googleads-php-lib\tests\Google\AdsApi\Common\SoapLogMessageFormatterTest.php:147
```

Maybe it's ok since it looks like a Windows thing:
`#Warning: Strings contain different line endings!`

### CODE REVIEW

**Comment by fiboknacky** [comment]
You've signed CLA already right?

**Comment by komirad** [comment]
Just did.

File: src/Google/AdsApi/AdWords/Testing/BatchJobs/v201609/BatchJobsTestProvider.php [hunk 1]
```diff
@@ -100,6 +100,15 @@ public static function getMutateResultArray() {
     $mutateResult2->setResult($operand);
     $mutateResult2->setIndex(1);
 
-    return [$mutateResult1, $mutateResult2];
+    $campaign = new FakeCampaign();
+    $campaign->setId(9223372036854775900);
+    $campaign->setName('Test large ID');
+    $operand = new FakeOperand();
+    $operand->setCampaign($campaign);
```
Comment (fiboknacky) [inline src/Google/AdsApi/AdWords/Testing/BatchJobs/v201609/BatchJobsTestProvider.php]: Can you put a new line after this line please?

Comment (komirad) [inline src/Google/AdsApi/AdWords/Testing/BatchJobs/v201609/BatchJobsTestProvider.php]: Ok, will figure out how I can edit the PR.

Comment (fiboknacky) [inline src/Google/AdsApi/AdWords/Testing/BatchJobs/v201609/BatchJobsTestProvider.php]: Never mind. I can do that when I sync this with our codebase.
Thanks for your contribution!

**Pull request merged**